Dockerfile

```diff
@@ -126,7 +126,7 @@ COPY ./Pipfile* ./
 #
 #   https://github.com/pypa/pipenv/issues/1226#issuecomment-598487793
 #
-ENV WORKON_HOME /simplified_venv
+ENV WORKON_HOME=/simplified_venv
```

Comment:
Does this update mean this wasn't working before or it's just a different syntax?

Reply:
It's only updated syntax, it should work as it did previously.